app/services/accredited_provider_api.rb

```diff
@@ -0,0 +1,20 @@
+class AccreditedProviderApi
+  include ServicePattern
+
+  def call
```

Comment:
Do we need to handle a 500 or any other response that it's not 200 here? What happens if publish is down?

Reply:
@CatalinVoineag I'm going to be following up with a more comprehensive implementation of the Provider API ([Trello card](https://trello.com/c/3iEk7hif/78-retrieve-providers-from-the-teacher-training-courses-api)) which will supersede this. But we went ahead with this implementation for now to avoid my card being a blocker.

It's worth noting that the response is cached, so we'll be somewhat protected from downtime just because we won't need to regularly hit the endpoint.

Reply:
Yeah, that makes sense. Thank you 👍🏻